Сделал второе задание. Работа с исключениями

CzechJD opened the pull request:
Там 4 коммита. Из-за невнимательности в первый раз добавил код с методом main.

CzechJD committed:
Задание с исключениями

CzechJD committed:
Задание с исключениями

CzechJD committed:
Задание с исключениями

CzechJD committed:
Задание с исключениями

psorokin02 commented on src/main/java/com/example/exceptions/exceptionBasics.java:
```diff
@@ -0,0 +1,25 @@
+package com.example.exceptions;
+
+public class exceptionBasics {
```
Класс всегда с большой буквы называть нужно, это такие договоренности в java-мире

psorokin02 commented on src/main/java/com/example/exceptions/tryWithResources.java:
```diff
@@ -0,0 +1,35 @@
+import java.io.*;
+import java.util.Scanner;
+
+public class tryWithResources {
```
Класс всегда с большой буквы называть нужно, это такие договоренности в java-мире

psorokin02 replied:
лучше создай тут main() и в этой функции выполни задание.
Сделай одну функцию, которая читает содержимое файла и вызвращает его из себя
И вторую функцию, которая на вход принимает текст и пишет его в выходной файл

psorokin02 commented on src/main/java/com/example/exceptions/tryWithResources.java:
```diff
@@ -0,0 +1,35 @@
+import java.util.Scanner;
+
+public class tryWithResources {
+    public static String readFile(String pathToFile) {
```
Не вижу, чтобы у тебя где-то этот метод вызывался. По заданию нужно прочитать текст из файла и записать в другой файл

CzechJD committed:
Merge branch 'master' into exceptions

CzechJD committed:
Изменены методы readFile() и writeInFile()
добавлены абсолютные пути для всех систем.
Добавлен метод main(), демонстрирующий работу программы
